jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java

```diff
@@ -757,4 +757,14 @@ public interface JpaLogger extends BasicLogger {
     @Message(id = 75, value="Illegal to call this method from injected, managed EntityManager")
     IllegalStateException illegalCallOnCloseMethod();
 
+    /**
+     * Logs warning detected a mix of bytecode enhanced + non-enhanced, will disable enhancement for all persistence units in this deployment.
+     * Also see discussion on https://hibernate.zulipchat.com/#narrow/channel/132094-hibernate-orm-dev/topic/ORM.20bytecode.20enhancement.20.2B.20WildFly if possible.
+     *
+     * @param scopedPuName name of the persistence unit
```

Comment:
I think the param should be scopedPuNames. So then

@param scopedPuNames comma-delimited names of the persistence units

Reply:
Change made locally and will be included in next push.